src/main/java/com/google/api/codegen/transformer/csharp/CSharpBasicPackageTransformer.java

```diff
@@ -41,6 +41,7 @@
       "csharp/gapic_snippets_csproj.snip";
   private static final String UNITTEST_CSPROJ_TEMPLATE_FILENAME =
       "csharp/gapic_unittest_csproj.snip";
+  private static final String SAMPLE_CSPROJ_TEMPLATE_FILENAME = "csharp/gapic_samples_csproj.snip";
```

Comment:
nit: for consistency, have this line and the previous formatted similarly

Reply:
Nope, formatter doesn't allow me to do that (so annoying!)